src/Sylius/Component/Product/Model/Product.php

```diff
@@ -351,6 +351,16 @@ public function getVariants()
     /**
      * {@inheritdoc}
      */
+    public function getAllVariants()
```

Comment:
This method name is not really true. It returns all variants that are not master ones, so it's not correct.